cmd/system-probe/main_common.go

```diff
@@ -100,9 +100,8 @@ func runAgent() {
 	log.Infof("system probe successfully started")
 
 	// Handles signals, which tells us whether we should exit.
-	e := make(chan struct{})
-	go util.HandleSignals(e)
-	<-e
+	go util.HandleSignals(exit)
```

Comment:
Handling signals here and allowing an exit channel to be passed in puts us in a weird position where we'll be doing shutdown logic  in two places. 

Can we move `go util.HandleSignals`  to main.go  and change the type of exit to `<- chan struct{}`, that way both the linux and windows versions are using the `exit` channel in the same way in both probes.

Reply:
Thanks for clarifying - I think I've changed it properly, but please let me know if I need to change things around still.